channels/fr.m3u

```diff
@@ -57,8 +57,6 @@ https://cdn02.hta.dz/abr_htatv/EL_HEDDAF_TV/htatv/EL_HEDDAF_TV_720p/chunks.m3u8
 https://emci-fr-hls.akamaized.net/hls/live/2007265/emcifrhls/04.m3u8
 #EXTINF:-1 tvg-id="EMCITVMontreal.fr" tvg-country="FR" tvg-language="" tvg-logo="" group-title="",EMCI TV Montreal (1080p)
 https://emci-td-hls.akamaized.net/hls/live/2007264/emcitdhls/index.m3u8
-#EXTINF:-1 tvg-id="EuronewsAlbania.fr" tvg-country="AL" tvg-language="Albanian" tvg-logo="https://i.imgur.com/hTBah1z.jpg" group-title="News",Euronews Albania [Offline]
-http://51.195.88.12:4050/live/euronewsmobpp2/playlist.m3u8
```

Comment:
https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UChR-A__NS_C5kHDWj3PeAhw/live

Reply:
Thanks for the link, couldn't find that one.